apps/next/package.json

```diff
@@ -35,6 +35,7 @@
     "@sentry/nextjs": "7.7.0",
     "@stripe/react-stripe-js": "2.1.0",
     "@stripe/stripe-js": "^1.52.1",
+    "@tanstack/react-virtual": "^3.0.0-alpha.0",
```

Comment:
should we replace it with `"^3.0.0-alpha.54"` because we are actually using `alpha.54` in our project and this will provide clarity.

Reply:
Oh good catch @alantoa - we're on the earliest alpha, which is crazy

Reply:
Done! good catch.